### DISCUSSION TITLE
about export glb/gltf error

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#3119] [Q&A]
hello

i am trying to export "availableVariants" model!

but i cant..ㅠㅠ help me

i can't understnad.. 

refer to this examples
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
https://modelviewer.dev/examples/scenegraph/#exporterWithOptions

![캡처](https://user-images.githubusercontent.com/87469003/149307453-0dad8eb5-b07c-45a2-8a52-63251151cf35.PNG)

**Comment by elalish** [comment]
Please list a set of repro steps to get to those errors; I'm not sure what you're trying to do. The export buttons on the link you shared  seem to be  working fine.
